css/util.css

```diff
@@ -107,71 +129,71 @@
 
 /* margin top */
 .mt-1 {
-  margin: 1rem;
+  margin-top: 1rem;
 }
 
 .mt-2 {
-  margin: 2rem;
+  margin-top: 2rem;
 }
 
 .mt-3 {
-  margin: 3rem;
+  margin-top: 3rem;
 }
 
 .mt-4 {
-  margin: 4rem;
+  margin-top: 4rem;
 }
 
 /* margin right */
 .mr-1 {
-  margin: 1rem;
+  margin-right: 1rem;
 }
 
 .mr-2 {
-  margin: 2rem;
+  margin-right: 2rem;
 }
 
 .mr-3 {
-  margin: 3rem;
+  margin-right: 3rem;
 }
 
 .mr-4 {
-  margin: 4rem;
+  margin-right: 4rem;
 }
 
 /* margin bottom */
 .mb-1 {
-  margin: 1rem;
+  margin-bottom: 1rem;
 }
 
 .mb-2 {
-  margin: 2rem;
+  margin-bottom: 2rem;
 }
 
 .mb-3 {
-  margin: 3rem;
+  margin-bottom: 3rem;
 }
 
 .mb-4 {
-  margin: 4rem;
+  margin-bottom: 4rem;
 }
 
 /* margin left */
 
 .ml-1 {
-  margin: 1rem;
+  margin-left: 1rem;
 }
 
 .ml-2 {
-  margin: 2rem;
+  margin-left: 2rem;
 }
 
 .ml-3 {
-  margin: 3rem;
+  margin-left: 3rem;
 }
 
 .ml-4 {
```

Comment:
use good names (descriptive)

Reply:
Can you give an example?
I've used same names as bootstrap 5 for less confusion. i used this because people are quite familiar bootstrap 5 and with this naming convention.